templates/activity-stream/impression-stats/impression-stats.1.schema.json

```diff
@@ -0,0 +1,87 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "impression-stats",
+  "properties": {
+    "action": {
+      "type": "string",
+      "pattern": "^activity_stream_impression_stats$"
+    },
+    "client_id": {
+      "type": "string",
+      "pattern": "^n/a$"
+    },
+    "session_id": {
+      "type": "string",
+      "pattern": "^n/a$"
+    },
+    "impression_id": {
+      "type": "string",
+      "pattern": "^\\{[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}\\}$"
+    },
+    "addon_version": {
+      "type": "string"
+    },
+    "version": {
+      "type": "string"
+    },
+    "release_channel": {
+      "type": "string"
+    },
+    "locale": {
+      "type": "string"
+    },
+    "topic": {
+      "type": "string"
+    },
+    "source": {
+      "type": "string"
+    },
+    "page": {
+      "type": "string",
+      "enum": [ "about:newtab", "about:home", "about:welcome", "unknown" ]
+    },
+    "region": {
+      "type": "string"
+    },
+    "profile_creation_date": {
+      "type": "integer"
+    },
+    "user_prefs": {
+      "type": "integer"
+    },
+    "click": {
+      "type": "integer"
+    },
+    "block": {
+      "type": "integer"
+    },
+    "pocket": {
+      "type": "integer"
+    },
+    "tiles": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": "integer"
+          },
+          "pos": {
+            "type": "integer"
+          }
+        },
+        "required": [ "id" ]
+      }
+    }
+  },
+  "required": [
```

Comment:
There are a few more fields that are not listed in the `data-event.md`.

* `shield_id`, varchar(256). A semicolon separated string to store a list of Shield IDs. 
* `hour`, integer
* `minute`, integer
* `sample_id`, server side generated integer ranges from 0 - 99
* `client_region`, varchar(5). Similar to `country_code`, but collected from the client side

Note that `hour` and `minute` are extracted from the timestamp field `receive_at`, which records the timestamp when Onyx receives the ping. Those two fields were requested by the Pocket team, need to confirm with them to see if they are still in use.

Reply:
Added these other than hour and minute.